front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/_shared/EmptyState.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { Box, Title, useBreakpoint } from '@citizenlab/cl2-component-library';
+import { useLocation } from 'react-router-dom';
+import { Multiloc } from 'typings';
+
+import useLocalize from 'hooks/useLocalize';
+
+import { DEFAULT_PADDING } from 'components/admin/ContentBuilder/constants';
+import Warning from 'components/UI/Warning';
+
+import { MessageDescriptor, useIntl } from 'utils/cl-intl';
+
+const isHomepageBuilder = (pathname: string) => {
+  return pathname.includes('/homepage-builder');
+};
+
+interface Props {
+  titleMultiloc: Multiloc;
+  explanation: MessageDescriptor;
+}
+
+// Only show this if we are in homepage builder
+const EmptyState = ({ titleMultiloc, explanation }: Props) => {
```

Comment:
Added 'empty state' that we only show in the homepage builder if there are no projects. On the real homepage, the widget would just be hidden if there are no projects. But of course this looks kind of strange in the homepage builder.